package.json

```diff
@@ -12,7 +12,8 @@
     "dist:rollup": "cross-env NODE_ENV=production rollup -c",
     "dist:transpile:cjs": "cross-env BABEL_MODULE=commonjs babel src --out-dir lib --extensions '.js,.ts,.jsx,.tsx'",
     "dist:transpile:esm": "babel src --out-dir lib-esm --extensions '.js,.ts,.jsx,.tsx'",
-    "dist:types": "tsc",
+    "dist:types:cjs": "tsc",
+    "dist:types:esm": "tsc --outDir lib-esm",
```

Comment:
Instead of re-running `tsc` can't we just copy the `.d.ts` file(s)?

Reply:
Also cross-platform copy possibly requires a new `npm` package? 😬